### ISSUE TITLE
"npm run dev" watch process does not work, at least for css and scss. URGENT!

### ISSUE THREAD

**Issue by andreibarbuta** [wprig/wprig#304]
When it runs it processes files for example css, but when it watches nothing happens.

**Comment by andreibarbuta** [comment]
After some digging it seems that the issue is regarding ${rootPath} constant. For example if I edit the line:
export const assetsDir = `${rootPath}/assets`;
...to...
export const assetsDir = `./assets`;

Then watch process works and processes css and scss files.

This may be a Windows issues, I am not sure.

**Comment by andreibarbuta** [comment]
It affects php files also.

**Comment by mor10** [comment]
What version are you working with?

**Comment by andreibarbuta** [comment]
Version 2.0.0, fresh install yesterday

**Comment by mor10** [comment]
@andreibarbuta have you tested with the latest version (a lot of things have changed in the build process in the past month). Also, can you specify your system (OS, node, npm, gulp, composer versions, etc.)

**Comment by andreibarbuta** [comment]
I tested it today using Windows 10 Pro, Version: 1809. NODE v9.3.0, NPM 5.5.1 and 6.8.0, Composer: 1.6.5 and 1.8.4.
I downloaded latest wprig version 2 from the github page.
I will make time this weekend to create a Virtual Machine and test it there so that it will be a completely clean Windows system.

This seems to be a different error.

This was the error:
[16:11:25] The following tasks did not complete: default, watch
[16:11:25] Did you forget to signal async completion?
events.js:136
      throw er; // Unhandled 'error' event
      ^

Error: listen EACCES 0.0.0.0:3001
    at Object._errnoException (util.js:999:13)
    at _exceptionWithHostPort (util.js:1020:20)
    at Server.setupListenHandle [as _listen2] (net.js:1362:19)
    at listenInCluster (net.js:1420:12)
    at Server.listen (net.js:1508:7)
    at Object.startServer [as fn] (D:\LANDO\wp-content\themes\wprig\node_modules\browser-sync-ui\lib\async.js:123:39)
    at D:\LANDO\wp-content\themes\wprig\node_modules\browser-sync-ui\lib\UI.js:184:14
    at iterate (D:\LANDO\wp-content\themes\wprig\node_modules\async-each-series\index.js:8:5)
    at D:\LANDO\wp-content\themes\wprig\node_modules\async-each-series\index.js:16:16
    at D:\LANDO\wp-content\themes\wprig\node_modules\browser-sync-ui\lib\UI.js:192:13
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! wprig@2.0.0 dev: `gulp`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the wprig@2.0.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Maverick\AppData\Roaming\npm-cache\_logs\2019-02-28T14_11_25_523Z-debug.log

**Comment by kauaicreative** [comment]
Here is the fix that works for me. The issue is with windows reversed slashes.
```
// Root path is where npm run commands happen. Correct windows slashes.
export const rootPath = process.env.INIT_CWD.replace(/\\/g,"/");
```

**Comment by ataylorme** [comment]
Can you try the [`v2.0-beta1`](https://github.com/wprig/wprig/releases/tag/v2.0-beta.1) release? `rootPath` is now `process.cwd()` which should return the proper results.

**Comment by andreibarbuta** [comment]
I will test it this weekend or monday at the latest. Thx.

**Comment by tanmalik786** [comment]
I just got it downloaded again [after having made few failed attempts yesterday] BUT couldn't achieve any of the things like
1. to watch for the changes to css files
2. to have it read the configuration settings from `config.json`, except for `browserSync` section

I am sharing the code in `config.json`, as it may prove helpful
`
{
    "theme": {
    "slug": "ctsoltheme",
    "name": "CTSol Theme",
    "author": "CosmicTechSol"
  },
  "dev": {
    "browserSync": {
      "live": true,
      "proxyURL": "wprig.local"
    },
    "debug": {
      "styles": true,
      "scripts": false,
      "phpcs": true
    }
  },
  "export": {
    "compress": true,
    "generatePotFile": true,
    "filesToCopy": [
      "LICENSE",
      "readme.txt",
      "style.css",
      "screenshot.png"
    ]
  }
}
`
I was expecting it to create a theme folder `ctsoltheme` [which is the slug for my theme, as the documentation for the v2.0 states.

**Comment by ataylorme** [comment]
@andreibarbuta please try the latest `master` branch. Issue #493 was causing gulp `watch` to not work on Windows.

**Comment by mor10** [comment]
should be fixed in `v2.0`. Closing due to inactivity.
